### DISCUSSION TITLE
triggering an update in the preset op.

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#421] [Feature Requests] [answered]
Just been playing with the preset op.
This would be even more powerful if you could actually trigger an update of a preset from the sidebar for example.

<img width="316" alt="Screenshot 2022-11-29 at 12 04 26" src="https://user-images.githubusercontent.com/36418408/204524639-7c4d4d97-2cd1-4b22-b79a-7b57486e0cd3.png">

This way a user could set their own paramaters and lock them in.
Would be ace!

**Comment by steam0r** [comment, marked as answer]
hm, this would be lost on page-reload though so it does not really make that much sense, does it?

**Reply by marioepsley** [reply]
It still allows a user to create their own setups/iterations though.
For everyone that makes tools for people to play with using cables it would be useful.
(not sure if it's a quick thing to do.)

I've also been playing with saving this information in local storage.
More successfully though linking it to the location hash so the settings can be called up.

I'll show you a prototype on Wednesday :)

**Reply by steam0r** [reply]
we "fixed" this in the meetup by writing state to LocationHash, this way it's properly persisting during reloads and state can be shared.
